src/layout/Beta/DefaultLayout/DefaultLayout.tsx

```diff
@@ -30,7 +30,7 @@ const DefaultLayout = ({ children }: { children: React.ReactNode }) => {
       <a className="usa-skipnav" href="#main-content">
         Skip to main content
       </a>
-      <div className={styles.siteContainer}>
+      <div className={`${styles.siteContainer}, ${styles.defaultTheme}`}>
```

Comment:
There shouldn't be a comma between these 2 classes, this is affecting the siteContainer class not being applied correctly. apologies I might have typed this incorrectly when we were pairing!